run.py

```diff
@@ -51,7 +51,11 @@ def _run_multiprocess(self, graph):
             len(ptrs)))
         results = list()
         for ptr in ptrs:
+            start = time()
             th = ptr.GetValue()
+            end = time()
+            logger.debug('%%%%%%%%%% Event loop for graph {} run in {} seconds'.format(
```

Comment:
Could you remove the `%%%%%%%`? We have the `DEBUG` flag for this.

Reply:
Ah, and we don't want the measurement there. We need it from the start of the function to the end. So to measure the "inclusive selftime" of the function.